testing/netatalk/netatalk.initd

```diff
@@ -0,0 +1,21 @@
+#!/sbin/openrc-run
+
+name=netatalk
+command="/usr/sbin/$name"
+command_args="-F $netatalk_config"
```

Comment:
Runscripts should have sane defaults if possible, i.e. don’t require config file in /etc/conf.d. Also more common name for config file is `conffile`.